scripts/screener/screener.runner.ts

```diff
@@ -48,23 +48,15 @@ async function scheduleScreenerBuild(
     }),
   });
 
-  if (response.status !== 201) {
-    throw new Error(`Call to proxy failed: ${response.status}`);
+  if (response.status === 200) {
+    console.log('Skipping screener check');
   }
-  //checkUrl
-  return response.json().then(url => url);
-}
-
-async function notifyIntegration(payload: ScreenerProxyPayload) {
-  const fetchResponse = await fetch(environment.screener.proxyUri, {
-    method: 'post',
-    headers: { 'Content-Type': 'application/json' },
-    body: JSON.stringify(payload),
-  });
 
-  if (fetchResponse.status !== 200) {
-    throw new Error(`Notify integration failed: ${fetchResponse.status}`);
+  if (response.status !== 201 && response.status !== 200) {
+    throw new Error(`Call to proxy failed: ${response.status}`);
   }
+  //conclusion of the screener run
+  return response.json().then(conclusion => conclusion);
```

Comment:
do we know the shape of `response.json()` ? if yes can we type it properly to drastically improve maintenance ?

also not sure what `conclusion` data shape means , but that could be addressed by adding that types. lemme know if you need help with adding that type. ty

Reply:
we also need to annotate `async function scheduleScreenerBuild` return of this function, because as now it return `Promise<any>` which is not good

Reply:
^ can be done as separate PR

Reply:
Agree with @Hotell there. I briefly checked code and it looks that it could something like that:

```ts
// Please correct the shape if needed
type ScheduleScreenerBuildResponse = {
  status?: 'completed'
  conclusion?: 'skipped' | 'failure' | 'in_progress' | 'cancelled'
  url?: string
}

async function scheduleScreenerBuild(/** ---- **/): Promise<ScheduleScreenerBuildResponse> {
// ---
// ---
  return await response.json() as ScheduleScreenerBuildResponse;
}
```

Reply:
yes, the shape of `response.json()` is defined within the screener proxy, as well as the `conclusion`, which can have the values `skipped`, `in_progress` that we addressed in another comment. This return value is again for logging purposes and I can just make the function to not return it anymore if needed

Reply:
Let's keep the value, but type it 👍